projects/rulesets-base/src/custom-rulesets/prepare-rulesets.ts

```diff
@@ -0,0 +1,91 @@
+import { Ruleset } from '../rules/ruleset';
+import { downloadRuleset } from './download-ruleset';
+import { resolveRuleset } from './resolve-ruleset';
+
+export type RulesetDef = {
+  name: string;
+  config: unknown;
+};
+
+export type RulesetPayload = {
+  ruleset: RulesetDef[];
+  localRulesets: Record<string, string>;
+  hostedRulesets: Record<
+    string,
+    {
+      uploaded_at: string;
+      url: string;
+    }
+  >;
+  standardRulesets: Record<
+    string,
+    { fromOpticConfig: (config: unknown) => Ruleset | string }
+  >;
+};
```

Comment:
Couple of changes from the old version:
- local rulesets can be defined and will just be resolved
- standad rulesets are injected (to avoid circular deps since this is in ruleset-base)
- hosted rulesets also include uploaded_at to allow for caching

Reply:
Is this updated in what goes over to rules orchestrator?

Reply:
Yep, I updated rules-orchestrator to include these changes where localrulesets is {} and we're matching the shapes